MAUI/NavigationDrawer/Swipe-Gesture.md

```diff
@@ -39,4 +39,35 @@ this.Content = navigationDrawer;
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Swipe Sensitivity
+
+The TouchThreshold property in the [SfNavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.html) can expand the swipe region.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings TouchThreshold="80">
```

Comment:
give threshold higher than the default value

Reply:
Yes, provided a higher value.